src/utils.py

```diff
@@ -93,6 +91,11 @@ def _pick(name, pick_list, default_file=None, helpfn=None, message='please pick:
         return choice
 
 def uin(param, default=0xDEADBEEF):
```

Comment:
trace what else is using this, no surprises please.

Reply:
surprisingly little actually. `buildvars.switch_revision`, `cfn.check_user_input` and `project.new`.

`cfn.check_user_input` is used to fetch an `instance_id` if one isn't provided when generating a new stack and probably shouldn't be buried so deep. It has a default of a datestamp.